backend/apps/accounts/throttles.py

```
@@ -10,48 +10,215 @@
+
+        if not redis_client:
+            raise RuntimeError("Redis connection unavailable for Lua rate limiting")
+
+        if _LUA_SHA is None:
+            _LUA_SHA = redis_client.script_load(LUA_RATE_LIMIT_SCRIPT)
+
+        try:
+            res = redis_client.evalsha(
+                _LUA_SHA, 1, self.key, self.window_seconds, self.max_requests
+            )
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)